tests/src/kinClient.intg.test.ts

```diff
@@ -0,0 +1,144 @@
+import {Environment} from "../../scripts/src/environment";
+import {KeyPair, KinClient} from "../../scripts/src";
+import {TransactionBuilder} from "../../scripts/src/blockchain/transactionBuilder";
+
+const keypair = KeyPair.generate();
+const seconedKeypair = KeyPair.generate();
+let client: KinClient;
+let account: KinAccount;
+let account2: KinAccount;
+
+describe("KinClient", async () => {
+	beforeAll(async () => {
+		client = new KinClient(Environment.Testnet);
+		account = client.createKinAccount({seed: keypair.seed});
+		account2 = client.createKinAccount({seed: seconedKeypair.seed});
+		const transactionId = await client.friendbot({address: keypair.publicAddress, amount: 10000});
+		const secondtransactionId = await client.friendbot({address: seconedKeypair.publicAddress, amount: 10000});
+		expect(transactionId).toBeDefined();
+		expect(secondtransactionId).toBeDefined();
+	}, 30000);
+
+	test("Create account with friend bot", async () => {
+		const data = await client.getAccountData(keypair.publicAddress);
+
+		expect(await client.isAccountExisting(keypair.publicAddress)).toBe(true);
+		expect(data.balances[0].balance).toBe(10000);
+	}, 30000);
+
+	test("Test getAccountData", async () => {
+		expect(await client.isAccountExisting(keypair.publicAddress)).toBe(true);
+		const data = await client.getAccountData(keypair.publicAddress);
+		expect(data.balances[0].balance).toBe(10000);
+		expect(data.balances.length).toBe(1);
+		expect(data.balances[0].assetType).toBe('native');
+		expect(data.signers.length).toBe(1);
+		expect(data.signers[0].publicKey).toBe(keypair.publicAddress);
+		expect(data.id).toBe(keypair.publicAddress);
+
+		const thirdKeypair = KeyPair.generate();
+		const builder = await account.buildCreateAccount({
+			address: thirdKeypair.publicAddress,
+			fee: 100,
+			startingBalance: 1000,
+			memoText: 'my first wallet'
+		});
+
+		await account.submitTransaction(builder);
+		const data2 = await client.getAccountData(keypair.publicAddress);
+		expect(data2.sequenceNumber).toBe(data.sequenceNumber + 1);
+	}, 30000);
+
+	test("Test getMinimumFee", async () => {
+		const thirdKeypair = KeyPair.generate();
+		const builder = await account.buildCreateAccount({
+			address: thirdKeypair.publicAddress,
+			fee: 100,
+			startingBalance: 1000,
+			memoText: 'my first wallet'
+		});
+
+		await account.submitTransaction(builder);
+		expect(await client.isAccountExisting(thirdKeypair.publicAddress)).toBe(true);
+		expect(await client.getMinimumFee()).toBe(100);
+	}, 30000);
+
+	test("Test getTransactionData", async () => {
+		const thirdKeypair = KeyPair.generate();
+		const builder = await account.buildCreateAccount({
+			address: thirdKeypair.publicAddress,
+			fee: 100,
+			startingBalance: 1000,
+			memoText: 'my first wallet'
+		});
+
+		const transactionId = await account.submitTransaction(builder);
+		const data = await client.getTransactionData(transactionId);
+		expect(data.fee).toBe(100);
```

Comment:
test all other fields of the transaction, any transaction will have a fee of 100.